### PR TITLE
NEW: global input settings editor (ISX-1474 )

### PR DESCRIPTION
**Pull request by ritamerkl**
### Description

provided methods to save, edit and load the Global Actions Asset via Input settings. According to this [ticket](https://jira.unity3d.com/browse/ISX-1474).

### Changes made

- added load functionality
- added save functionality (integrated in refactored old save method)

### Checklist

Before review:

- [ ] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - FogBugz ticket attached, example `([case %number%](https://issuetracker.unity3d.com/issues/...))`.
    - FogBugz is marked as "Resolved" with *next* release version correctly set.
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [ ] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public class InputActionsEditorWindowUtils
+    {
+        public static StyleSheet theme => EditorGUIUtility.isProSkin
+        ? AssetDatabase.LoadAssetAtPath<StyleSheet>(InputActionsEditorConstants.PackagePath + InputActionsEditorConstants.ResourcesPath + "/InputAssetEditorDark.uss")
+        : AssetDatabase.LoadAssetAtPath<StyleSheet>(InputActionsEditorConstants.PackagePath + InputActionsEditorConstants.ResourcesPath + "/InputAssetEditorLight.uss");
+
+        public static void SaveAsset(SerializedObject serializedAsset)
+        {
+            var asset = (InputActionAsset)serializedAsset.targetObject;
+            if (asset.name == HighLevel.Input.kGlobalActionsAssetName)
```
Comment (jamesmcgill) [inline Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs:23]: Maybe add some comments here to remind your future self what is going on here (that it's handling 2 cases)

File: Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs [hunk 2]
```diff
@@ -1,13 +1,21 @@
 #if UNITY_EDITOR && UNITY_INPUT_SYSTEM_UI_TK_ASSET_EDITOR // We use some UITK controls that are only available in 2022.2 or later (MultiColumnListView for example)
+using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```
Comment (jamesmcgill) [inline Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs]: Are these additional `using`s actually used (e.g. Linq)

**Review by jamesmcgill — APPROVED** [review]
Looks good

File: Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs (hunk 2 above)
Comment (ritamerkl) [reply Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs]: leftovers, I removed them

**Pull request merged**